packages/taquito/src/operations/types.ts

```diff
@@ -76,6 +76,27 @@ export type RPCOpWithSource =
   | RPCSmartRollupAddMessagesOperation
   | RPCSmartRollupOriginateOperation;
 
+export const isOpWithGasBuffer = <
```

Comment:
do you mind explaining a bit what this method does?

Reply:
from my research that octez-client only add 100 gas buffer to these operation `[update_consensus_key, register_global_constants, transfer_ticket, originate, contract_call, smart_rollup_originate, smart_rollup_add_message]`
gasLimit will correlated of fee calculation so being precise of which ops to add which doesn't help lower fee